app/models/importers/organisation.rb

```diff
@@ -25,7 +24,7 @@ def run
     raise 'No result for slug' if organisation.content_items.empty?
   end
 
-  private
+private
 
```

Comment:
I just fixed all lint errors in one commit. I prefer not to put time into fixing up this line. I hope this is ok with you.